auslib/blobs/schemas/systemaddons.yml

```diff
@@ -0,0 +1,71 @@
+---
+  title: GMP Blob
+  type: object
+  required:
+    - name
+    - schema_version
+    - hashFunction
+    - vendors
+  additionalProperties: false
+  properties:
+    name:
+      type: string
+      description: Human readable identifier
+      minLength: 1
+      maxLength: 100
+    schema_version:
+      type: number
+      description: Blob identifier number
+      enum:
+        - 1000
+    hashFunction:
+      type: string
+      description: Hash function of files pointed at (returned in responses generated by this Blob)
+    uninstall:
+      type: boolean
+      description: If this is set true, all system addons are disabled
+      default: false
+    vendors:
```

Comment:
I think it would be good to make sure "uninstall" and "vendors" can't be set at the same time. It's not obvious what should happen if both are present, so it's best to make sure that can't happen.